src/platform/packages/shared/kbn-field-types/src/kbn_field_types_factory.ts

```diff
@@ -121,6 +121,11 @@ export const createKbnFieldTypes = (): KbnFieldType[] => [
     filterable: true,
     esTypes: [ES_FIELD_TYPES.HISTOGRAM],
   }),
+  new KbnFieldType({
+    name: KBN_FIELD_TYPES.EXPONENTIAL_HISTOGRAM,
+    filterable: true,
```

Comment:
@jughosta what this flag is doing?

Reply:
Do you want to set it to false because you don't see it or because you think it's incorrect?
The _Basic_ Histogram is filterable

Reply:
Agree, let's set it as `false` then.

Found 2 issues where it does not work:
<img width="752" height="617" alt="Screenshot 2025-11-13 at 10 02 51" src="https://github.com/user-attachments/assets/c3fbb285-e74d-490f-8344-93a064399d4f" />
<img width="1263" height="452" alt="Screenshot 2025-11-13 at 10 03 54" src="https://github.com/user-attachments/assets/9f510756-a268-484a-abf7-a7447a896601" />